src/vehicle.cpp

```diff
@@ -3116,30 +3043,22 @@ point vehicle::coord_translate( const point &p ) const
 }
 
 void vehicle::coord_translate( const units::angle &dir, const point &pivot, const point &p,
-                               tripoint &q ) const
+                               tripoint &q )
```

Comment:
Why did you remove the const on these two functions?

Reply:
Because you made them static, okay.  That makes sense.

Reply:
I'll revert this change to vehicle::coord_translate. It's extra churn and the lack of "const" throws people off.
It was needed early in development, but now I have full access to all of the vehicle functions.

Reply:
I could go either way on this.  There's no obvious reason why either these functions need to be repeated per vehicle instance, and arguably (but I'll leave that up to jbtw) they should in the tripoint or point classes.

Reply:
Ping jbytheway?

Reply:
For a PR with so much else going on I'd say don't change it if there's no particular reason to.  It will probably have to change once my point type refactoring effort reaches vehicle coordinates, but that's probably many months away.  I lean towards them logically fitting in the `vehicle` class more than `point`/`tripoint`.  `point` is aiming to be pretty minimal.

Reply:
> For a PR with so much else going on I'd say don't change it if there's no particular reason to. It will probably have to change once my point type refactoring effort reaches vehicle coordinates, but that's probably many months away. I lean towards them logically fitting in the `vehicle` class more than `point`/`tripoint`. `point` is aiming to be pretty focused on it's primary purpose.